experiments/generic/node_memory_hog/README.md

```diff
@@ -0,0 +1,14 @@
+## Experiment Metadata
```

Comment:
Why is the use of the above binary file, experiments/generic/node_memory_hog/.node_memory_hog_k8s_job.yml.swp